changelog.txt

```diff
@@ -1,3 +1,30 @@
+14.0 (2024-04-13)
+------------------------------------------------------------------------
+Update: New title game for 14.0
+Fix #12477: Use std::filesystem::rename instead of Windows Shell API call (#12478)
+Fix #12233: Mini order list overlaps vehicle group name (#12423)
+Fix #12114: Viewport coords of crashed aircraft not updated when falling (#12424)
+Fix #12395: Ensure president name widget is tall enough (#12419)
+Fix #12415: Aircraft::tile is valid only for front vehicle (#12416)
+Fix #12387: [NewGRF] Wrong tile offset passed to rail station CB 149 (slope check)
+Fix #12388: Vehicle::CopyVehicleConfigAndStatistics not releasing unit number (#12389)
+Fix #12268: Capitalize "Wait to unbunch" order string (#12375)
+Fix #12368: Incorrect offset for click position within industry chain window (#12370)
+Fix: Signature validation did not close its file (#12479)
+Fix: Aircraft crash counter was too low to reach ground (#12425)
+Fix: Segfault when using -q without providing a . character (#12418)
+Fix: Wrong scrolling dropdown list position with RTL (#12412)
+Fix: [Win32] Force font mapper to only use TrueType fonts (#12406)
+Fix: Server shutdown and newgame packets should be stable (#12379)
```

Comment:
Remove, if you ask me. It is a new agreement we have on protocol level, no user minds, I would say.